docs/conf.py

```diff
@@ -232,7 +232,7 @@
 
 
 def setup(app):
-    app.add_stylesheet("theme_overrides.css")
+    app.add_css_file("theme_overrides.css")
```

Comment:
Apparently, the Sphinx API has changed and this update was required … glad I build the docs!